requirements/install.in

```diff
@@ -22,7 +22,7 @@ Flask-Mail==0.9.1
 flask-mongoengine==0.9.5
 Flask-Navigation==0.2.0
 flask-restx==1.0.5
-Flask-Security-Too==4.1.5
+Flask-Security-Too==5.1.1
```

Comment:
5.1.2 is out already ! :p
With a bit of additional doc on customizing that may be useful: https://github.com/Flask-Middleware/flask-security/pull/773/files